src/profile-logic/import/chrome.js

```diff
@@ -918,28 +927,34 @@ function extractMarkers(
             category: event.cat,
             data: argData,
           });
-        } else if (event.ph === 'B' || event.ph === 'E') {
-          if (event.ph === 'B') {
-            // The 'B' phase stand for "begin", and is the Chrome equivalent of IntervalStart.
+        } else if (
+          event.ph === 'B' ||
+          event.ph === 'E' ||
+          event.ph === 'b' ||
+          event.ph === 'e'
+        ) {
+          if (event.ph === 'B' || event.ph === 'b') {
+            // The 'B' and 'b' phases stand for "begin", and is the Chrome equivalent of IntervalStart.
             markers.startTime.push(time);
             markers.endTime.push(null);
             markers.phase.push(INTERVAL_START);
           } else {
-            // The 'E' phase stand for "end", and is the Chrome equivalent of IntervalEnd.
+            // The 'E' and 'e' phase stand for "end", and is the Chrome equivalent of IntervalEnd.
             markers.startTime.push(null);
             markers.endTime.push(time);
             markers.phase.push(INTERVAL_END);
           }
 
-          // Duration Event
+          // Duration or Async Event
           // https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.nso4gcezn7n1
           markers.data.push({
             type: 'tracing',
             category: event.cat,
             data: argData,
           });
         } else {
-          // This assumes the phase is 'I', or Instant.
+          // This assumes the phase is 'I' or 'i' (Instant), 'n' (Async Instant)
+          // or 'R' (Mark events)
```

Comment:
I feel like this if if statement is getting bigger and probably will get bigger in the future in case they add a new type. I wonder if it makes more sense to have an extensive check here instead of an else block. But also it was like this before your patch, so I don't think it's worth blocking for it.

Reply:
Yeah, I think it would be clearer to do a big switch/case here. I believe we want to avoid doing the initial code (getThreadInfo etc) when not needed but maybe that's not a big deal...